cvat/apps/analytics_report/report/get.py

```diff
@@ -0,0 +1,127 @@
+# Copyright (C) 2023 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from datetime import datetime, timedelta, timezone
+from dateutil import parser
+
+from rest_framework import serializers, status
+from rest_framework.response import Response
+
+from cvat.apps.engine.models import Job, Task, Project
+from cvat.apps.analytics_report.models import TypeChoice, AnalyticsReport
+from cvat.apps.analytics_report.serializers import AnalyticsReportSerializer
+
+
+def _filter_statistics_by_date(statistics, start_date, end_date):
+    for _, st_entry in statistics.items():
+        dataseries = st_entry.get("dataseries", {})
+        for ds_name, ds_entry in dataseries.items():
+            dataseries[ds_name] = list(filter(lambda df: start_date <= parser.parse(df["datetime"]) <= end_date, ds_entry))
+
+    return statistics
+
+def _convert_datetime_to_date(statistics):
+    for st_entry in statistics.values():
+        dataseries = st_entry.get("dataseries", {})
+        for ds_entry in dataseries.values():
+            for df in ds_entry:
+                df["date"] = parser.parse(df["datetime"]).date()
+                del df["datetime"]
+    return statistics
+
+def _get_object_report(obj_model, pk, start_date, end_date):
+    db_obj = obj_model.objects.get(pk=pk)
+    try:
+        db_analytics_report = db_obj.analytics_report
+    except AnalyticsReport.DoesNotExist:
+        raise
```

Comment:
Probably, there is no point in this try-catch block. I think it would be reasonable to convert doesn't exist errors (both from reports and `obj`) to `rest_framework.exceptions.NotFoundNotFound` for the response.